src/pages/iou/WaypointEditor.js

```diff
@@ -161,7 +163,7 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
             onEntryTransitionEnd={() => textInput.current && textInput.current.focus()}
             shouldEnableMaxHeight
         >
-            <FullPageNotFoundView shouldShow={Number.isNaN(parsedWaypointIndex) || parsedWaypointIndex < 0 || parsedWaypointIndex > waypointCount - 1}>
```

Comment:
@gegham-khachatryan Can you please help explain why do we need to use `&& isFocused` here? Thanks

Reply:
@ahmedGaber93  There was an issue where when deleting a waypoint, transactions were updated before the page close animation occurred, and the Not Found screen became visible for a few milliseconds during the animation.